ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/mpls-interfaces.j2

```diff
@@ -3,19 +3,29 @@
 
 ## MPLS Interfaces
 
-| Interface | MPLS IP Enabled | LDP Enabled |
-| --------- | --------------- | ----------- |
+| Interface | MPLS IP Enabled | LDP Enabled | IPv4 Node-Segment Index | IPv6 Node-Segment Index |
```

Comment:
Is MPLS interfaces the right place for this? Alternative could be ISIS interface table or a new table. Not sure.

Reply:
Moved to ISIS documentation.